pyproject.toml

```diff
@@ -20,7 +20,7 @@
 
 [tool.poetry]
 name = "renku"
-version = "0.0.0" # placeholder, see poetry-dynamic-versioning
+version = "1.0.0.dev3+g2833b272" # placeholder, see poetry-dynamic-versioning
```

Comment:
Should we keep `0.0.0` format or it's not important?

Reply:
Well it's not important but cleaner if it's 0.0.0, good catch :)